eclide/EclDlgBuilder.cpp

```diff
@@ -748,6 +748,11 @@ void CBuilderDlg::OnEclGo(UINT /*uNotifyCode*/, int nID, HWND /*hWnd*/)
     case ID_GO_SUBMITSELECTED:
         m_owner->OnButtonGo(Dali::WUActionRun, true);
         break;
+    case ID_ECL_NOARCHIV_GO:
+    case ID_GO_SUBMITNOARCHIV:
+        m_archive = false;
```

Comment:
Should this be set back to true after submission, or does that happen elsewhere?

Reply:
@dehilsterlexis - did you confirm this eitherway?